3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/pii/PiiPseudonymizerEnrichmentSpec.scala

```diff
@@ -135,8 +137,9 @@ class PiiPseudonymizerEnrichmentSpec extends Specification with ValidationMatche
     val input: ValidatedMaybeCollectorPayload = Some(collectorPayload).successNel
     val rrc                                   = new RepositoryRefConfig("test-schema", 1, List("com.snowplowanalytics.snowplow"))
     val repos                                 = TestResourcesRepositoryRef(rrc, "src/test/resources/iglu-schemas")
+    val adapterRegistry                       = new AdapterRegistry(Map.empty[(String, String), RemoteAdapter])
```

Comment:
No need to specify `Map.empty[(String, String), RemoteAdapter]`.

Reply:
done